cypress/support/commands.js

```diff
@@ -44,3 +44,12 @@ Cypress.Commands.add("assertClipboardValue", (value) => {
 Cypress.Commands.add("getBySelector", (selector, ...args) => {
   return cy.get(`[data-cy=${selector}]`, ...args);
 });
+
+Cypress.Commands.add("blockAnnouncements", () => {
```

Comment:
This seems like it would need to be applied to every test in order to avoid it interrupting them. Can we apply it by default and then manual exclude if needing to test the announcement behavior? This way we don't have to remember to add it for every new test.